scripts/setup.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 set -xeuo pipefail
 
+# Directory containing terraform scripts
+[[ -z ${TERRAFORM_WD:-} ]] && TERRAFORM_WD=$(pwd)/terraform
```

Comment:
2 fixes:

- logic error where we are binding this variable too late
- syntax error where we need to surround the variable check with `{}`

Reply:
Just curious: what does `:-` accomplish here? Would plain `${TERRAFORM_WD}` not evaluate the same way?